website/src/hugo/content/changelog.md

```diff
@@ -46,17 +46,21 @@ it.
 * Move `org.http4s.server.ServerSoftware` to `org.http4s.ServerSoftware` [#1884](https://github.com/http4s/http4s/pull/1884)
 * Fix `Uncompressible` and `NotBinary` flags in `MimeDB` generator. [#1900](https://github.com/http4s/http4s/pull/1884)
 * Generalize `DefaultHead` middleware to work on `Http[F, G]` given `Functor[F]` and `MonoidK[F]` [#1903](https://github.com/http4s/http4s/pull/1903)
-* Generalize `GZip` middleware to work on `Http[F, G]` given `Functor[F]` and `Functor[G]` [#1903](https://github.com/http4s/http4s/pull/1903)
+* Generalize `GZip` middleware to work on `Http[F, G]` given `Functor[F]` and `Functor[G]` [#1903]
+(https://github.com/http4s/http4s/pull/1903)
+* Removed charset for argonaut instances [#1914](https://github.com/http4s/http4s/pull/1914)
```

Comment:
This change was listed in 0.18.13 but was merged against master. Do we want to backport this work?